src/main/java/org/opentripplanner/routing/algorithm/filterchain/api/GroupBySimilarity.java

```diff
@@ -1,6 +1,7 @@
-package org.opentripplanner.routing.algorithm.filterchain;
+package org.opentripplanner.routing.algorithm.filterchain.api;
 
 import org.opentripplanner.framework.tostring.ToStringBuilder;
+import org.opentripplanner.routing.algorithm.filterchain.ItineraryListFilterChainBuilder;
```

Comment:
This import is only used in the docstring. In ItinerarySortKey you changed the reference in the comment to the qualified path. Should you do the same here for consistency?

Reply:
We do not have a policy on this yet. But, maybe we should. There are a are forces:
 - Using imports have the negative effect that it looks like there is a dependency from a class file, which does not exist - and worse, it show up in static analyses of package dependencies. 
 - Using qualified paths is hard to read in the code.
 
 So, what I tend to do do is to import ok dependencies and use qualified path when the dependency is not ok. This is not so easy to follow, so my suggestion would be to always use qualified paths in doc - at leas if you are unsure. But, lets discuss this on todays meeting.